README.md

```diff
@@ -174,4 +174,4 @@ We currently have plans to support:
 
 This project is under an MIT license
 
-Powered by: [MagenTys](http://magentys.io)
+Powered by: [MagenTys](http://magentys.io) [MechanicalRock](https://www.mechanicalrock.io/#/)

```

Comment:
can we add an & or an AND in between the two companies.
The MagenTys site should also be https

Reply:
[Mechanical Rock](https://mechanicalrock.io)

Reply:
Done